definitions/npm/commander_v2.x.x/flow_>=0.28.x/commander_v2.x.x.js

```diff
@@ -0,0 +1,276 @@
+   * Allow unknown options on the command line.
+   *
+   * @param {Boolean} arg if `true` or omitted, no error will be thrown
+   * for unknown options.
+   * @api public
+   */
+  allowUnknownOption(arg?: boolean): $npm$commander$Command;
+
+  /**
+   * Parse `argv`, settings options and invoking commands when defined.
+   *
+   * @param {Array} argv
+   * @return {Command} for chaining
+   * @api public
+   */
+  parse(argv: Array<string>): $npm$commander$Command;
+
+  /**
+   * Parse options from `argv` returning `argv`
+   * void of these options.
+   *
+   * @param {Array} argv
+   * @return {Array}
+   * @api public
+   */
+  parseOptions(argv: Array<string>): { args: Array<string>, unknown: Array<string> };
+
+  /**
+   * Define argument syntax for the top-level command.
+   *
+   * @api public
+   */
+  arguments(desc: string): $npm$commander$Command;
+
+  /**
+   * Return an object containing options as key-value pairs
+   *
+   * @return {Object}
+   * @api public
+   */
+  opts(): any;
+
+  /**
+   * Set the program version to `str`.
+   *
+   * This method auto-registers the "-V, --version" flag
+   * which will print the version number when passed.
+   *
+   * @param {String} str
+   * @param {String} flags
+   * @return {Command} for chaining
+   * @api public
+   */
+  version(str: string, flags?: string): $npm$commander$Command;
+
+  /**
+   * Set the description to `str`.
+   *
+   * @param {String} str
+   * @return {String|Command}
+   * @api public
+   */
+  description(str: string): $npm$commander$Command;
+  description(): string;
+
+  /**
+   * Set an alias for the command
+   *
+   * @param {String} alias
+   * @return {String|Command}
+   * @api public
+   */
+  alias(alias: string): $npm$commander$Command;
+  alias(): string;
+
+  /**
+   * Set / get the command usage `str`.
+   *
+   * @param {String} str
+   * @return {String|Command}
+   * @api public
+   */
+  usage(str: string): $npm$commander$Command;
+  usage(): string;
+
+  /**
+   * Get the name of the command
+   *
+   * @param {String} name
+   * @return {String|Command}
+   * @api public
+   */
+  name(): string;
+
+  /**
+   * Output help information for this command
+   *
+   * @api public
+   */
+  outputHelp(): void;
+
+  /**
+   * Output help information and exit.
+   *
+   * @api public
+   */
+  help(): void;
+}
+
+declare class $npm$commander$Option {
+  /**
+   * Initialize a new `Option` with the given `flags` and `description`.
+   *
+   * @param {String} flags
+   * @param {String} description
+   * @api public
+   */
+  constructor(flags: string, description?: string): $npm$commander$Option;
+  flags: string;
+  required: boolean;
+  optional: boolean;
+  bool: boolean;
+  short?: string;
+  long: string;
+  description: string;
+}
+
+type $npm$commander$exports = $npm$commander$Command & {
+  Command: $npm$commander$Command,
+  Option: $npm$commander$Option
+};
+
+declare module "commander" {
+  declare var exports: $npm$commander$exports;
```

Comment:
`declare var exports` is now deprecated in favor of `declare module.exports: `

I'm waiting a few more versions before I start emitting a deprecation notice here (I want to give the syntax some time to work its way into history a bit), but since this libdef is designated `>=0.28.0` it's good to start using the new syntax.